index.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+const assert = require('assert');
+
+class DejaVu {
+
+  /**
+   * Constructs an new instance of DejaVu to identify "original" events.
+   * @param {Object} options The constructor options.
+   * @param {Object} options.redisConnection A connection to a Redis deployment.
+   */
+  constructor(options) {
+    assert(options.redisConnection, 'Must provide a Redis connection');
+    this._redisConnection = options.redisConnection;
+    this._handlers = {};
+  }
+
+  /**
+   * isNew returns if an event is new or not.
+   * @param {String} prefix The prefix to namespace events under.
+   * @param {Number} window The window to filtet event timestamps by (in ms).
```

Comment:
Can we call this `timeLimit`? A bit more explicit, also less confusing in JavaScript (I know this module is Node-only and likely to remain that way given its use of Redis, but GitHub syntax highlighting is still confused)